test/bmc/integration_test.rb

```diff
@@ -21,7 +23,23 @@ def test_features
     mod = response['bmc']
     refute_nil(mod)
     assert_equal('running', mod['state'], Proxy::LogBuffer::Buffer.instance.info[:failed_modules][:bmc])
-    assert_equal([], mod['capabilities'])
+    assert_equal(['redfish', 'shell', 'ssh'], mod['capabilities'])
```

Comment:
This could use the frozen constant too.

Reply:
are frozen constants really that useful in tests?